packages/formatters/src/formatters.ts

```diff
@@ -373,6 +373,87 @@ export function messageLink<C extends Snowflake, M extends Snowflake, G extends
 	return `${guildId === undefined ? channelLink(channelId) : channelLink(channelId, guildId)}/${messageId}`;
 }
 
+export enum HeadingLevel {
```

Comment:
I'd prefer if there were some basic documentation for this. Something like "Heading levels used for expanded markdown" and then "The first heading level", "The second heading level", etc. Otherwise, on the documentation website, it might show up blank. Basic documentation is better than nothing.

Reply:
I didn't think about that, thank you for letting me know! I've made the changes.